spanner/google/cloud/spanner_v1/keyset.py

```diff
@@ -60,16 +65,14 @@ def to_pb(self):
         """
         kwargs = {}
 
-        if self.start_open:
+        if self.start_open is not None:
             kwargs['start_open'] = _make_list_value_pb(self.start_open)
-
-        if self.start_closed:
+        else:
```

Comment:
@chemelnucfin As I mentioned before, anything other than the state it was in before is just re-validating the value. The only change this method needed was converting the `if self.foo:` checks into `if self.foo is not None:` checks.